<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DummyDataCreator</class>
 <widget class="QDialog" name="DummyDataCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Report Template / Dummy Data</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Instance counts:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Generate report template</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>close</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Template presets:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="5" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Pagefooter Image Path:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Report Title:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLineEdit" name="edt_title"/>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="edt_image_header"/>
      </item>
      <item row="6" column="0">
       <widget class="QLineEdit" name="edt_image_footer"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Pageheader Image Path:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QToolButton" name="btn_image_header">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QToolButton" name="btn_image_footer">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
